sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -183,7 +183,7 @@ def extract_data(pipeline_response):
             link = None
             if "Link" in pipeline_response.http_response.headers.keys():
                 link = _parse_next_link(pipeline_response.http_response.headers["Link"])
-            elif "link" in pipeline_response.http_response.headers.keys():  # python 2.7 turns this into lowercase
+            elif "link" in pipeline_response.http_response.headers.keys():
```

Comment:
We shall either remove the if, or leave a comment "This test was for Py2 only and we should consider to remove it". IF you just remove the comment, in 6 months we will not remember why this test is here at all :)

Reply:
Removed the elif section for python 2.7